### ISSUE TITLE
Very bad news ! 09.11.2018_08:55 Login error! Connection error!    

### ISSUE THREAD

**Issue by AmauryBJ** [instabot-py/instabot.py#1723]
Very bad news

Since this morning.
Impossible to connect and i get a Suspisious connectiong on instragram on my phone.

They probably create a better system to check the connection location.

**Comment by OlivierMrz** [comment]
Mine works fine!

**Comment by Tschingis0711** [comment]
Same here!! Someone an effective solution?

**Comment by oakleyblack** [comment]
yeah, just tried logging in... same result. was also trying to wipe my posts and follows with a different service and couldn't verify my login. i think IG updated their security again.

**Comment by AmauryBJ** [comment]
> @AmauryBJ @Mehranp Mine works fine!

Because your server is in your home ! if you log at a different location from your server you will have the issue.

**Comment by Tschingis0711** [comment]
Guys, just made the changes mentioned in #1718 . Works for me. Check out.

**Comment by AmauryBJ** [comment]
> Guys, just made the changes mentioned in #1718 . Works for me. Check out.

Not link at all !

**Comment by oakleyblack** [comment]
@Tschingis0711 that didn't work for me

**Comment by w3x10e8** [comment]
Mine was working fine yesterday, but today i wokeup and saw this error 

`    json_info = json.loads(re.search('{"activity.+probably_has_app', info.text, re.DOTALL).group(0)+'":""}')
AttributeError: 'NoneType' object has no attribute 'group'`

**Comment by AmauryBJ** [comment]
> Mine was working fine yesterday, but today i wokeup and saw this error
> 
> ` json_info = json.loads(re.search('{"activity.+probably_has_app', info.text, re.DOTALL).group(0)+'":""}') AttributeError: 'NoneType' object has no attribute 'group'`

You need to update to solve this error, and after that you will have the Login issue.

**Comment by oakleyblack** [comment]
Still no info/fix?

**Comment by sourstripe** [comment]
having a login error now too

**Comment by AmauryBJ** [comment]
You need to log with the bot on the same ip you use on your phone, after that the session will stay open, but if you try to reconnect the bot remotly from another location it will be identified as a suspicious connection and will be bloked..

**Comment by teylerk** [comment]
I am getting the error on my home IP and the IP has never changed. Getting on every login. One account currently working fine which makes me think there is a new sliding scale for suspicious login blocking. Let it sit for 24 hours and try again.

**Comment by Dissunio** [comment]
Some solution?

**Comment by Dissunio** [comment]
After switching on the bot you have to go to the instagram and confirm your location helped me.

**Comment by Martypary** [comment]
>           Can't get to work for anything now. None of my accounts can log in on any ip or proxy, gets a suspicious login.

I have the same : |

**Comment by AmauryBJ** [comment]
Nope no solutions ! It's the end ! Repent your sins guy !

**Comment by josamgeek** [comment]
I'm experiencing the same issue. 

**Szenario**: I tried restarting the bot (that was working fine before) this morning -> Couldn't login "Connection Error". I got a "suspicious Connection" message on my phone and approved that it was me. Retried -> Same thing. After some attempts I didn't get a notification on my phone anymore but the bot couldn't login still.

**Info that might help investigating the problem further:** 
- I am running several bots for different profiles on AWS.
- I stoped and restarted other bot and it worked fine.
- I can login to the account for which the bot is not working in my browser (also in private mode e.g. no cookies).
- If I run the bot for this account locally in the terminal -> It is not working (Both with default settings and also if I change the user agent to match my browser)

**Conclusion** form my side:
- Contrary to my initial idea it looks like this is not a problem of the IP that access.
- Sadly I have no clue what it is then... 

If anyone finds a fix. Please let us know.

**Comment by flite3** [comment]
One of my accounts was getting a connection error and unable to log in.  I went into the app and it gave me a notice that there was a suspicious login attempt.  I said yes this was me. I logged in with the script again and it started working fine, I was using the same IP address for my bot as before.  I then got an email from Instagram asking notifying me that an attempt from a computer that I don't usually use was made from my bots location.  It's checking our usage against our login habits it seems. If it's different it stops the login it seems.  Right now the bot is working, but if I logout and try again, I'm not sure it will let my bot in again.

**Comment by rberthoud** [comment]
I was having this issue for the past two days. Even after I confirmed the activity when prompted with the "suspicious login attempt" issue, IG wouldn't allow the bot to login.
I had basically given up, but today I tried to run the bot just to check it and it actually worked as if the login issue had never happened. I'm not sure if I got lucky or if the block was an undesired side effect of some update on IG security that they eventually fixed. Anyone else had the same luck? Can we trust Instabot will come back to life for good?

**Comment by dcr226** [comment]
This is a shame, I'm getting the same issue. Perhaps its the IG "Checkpoint" notification. I even tried logging in with a browser on the same machine running the bot..same issue occurs

**Comment by oakleyblack** [comment]
> I was having this issue for the past two days. Even after I confirmed the activity when prompted with the "suspicious login attempt" issue, IG wouldn't allow the bot to login.
> I had basically given up, but today I tried to run the bot just to check it and it actually worked as if the login issue had never happened. I'm not sure if I got lucky or if the block was an undesired side effect of some update on IG security that they eventually fixed. Anyone else had the same luck? Can we trust Instabot will come back to life for good?

My bot is logging in fine with no errors now as well... Weird. Still getting Not liked: 403 errors though. anyone have a fix?

**Comment by heldausberlin** [comment]
Does nobody else has this problem? I use mode 1. Who else?

**Comment by flite3** [comment]
I think this might explain our problem here. If you’ve been using this script for a while, you might have obscured your app habits from the ai, but new users will probably not be able to use this script moving forward I think.
https://9to5mac.com/2018/11/19/instagram-deleting-inauthentic-content/

**Comment by rberthoud** [comment]
> > I was having this issue for the past two days. Even after I confirmed the activity when prompted with the "suspicious login attempt" issue, IG wouldn't allow the bot to login.
> > I had basically given up, but today I tried to run the bot just to check it and it actually worked as if the login issue had never happened. I'm not sure if I got lucky or if the block was an undesired side effect of some update on IG security that they eventually fixed. Anyone else had the same luck? Can we trust Instabot will come back to life for good?
> 
> My bot is logging in fine with no errors now as well... Weird. Still getting Not liked: 403 errors though. anyone have a fix?

Today it woke up dead again x.x

**Comment by pierresauvignon** [comment]
Same here across 4 accounts :(
Verifying that "This was me" in the IG iOS app doesn't change anything.
Interestingly enough I can login fine in the browser... (same user agent and IP as instabot).

**Comment by rberthoud** [comment]
> Same here across 4 accounts :(
> Verifying that "This was me" in the IG iOS app doesn't change anything.
> Interestingly enough I can login fine in the browser... (same user agent and IP as instabot).

Now it doesn't even ask for verification on IG anymore... it's just plain access denial...

**Comment by ggunay** [comment]
Has anybody got a fix for that? seems like still not working

**Comment by ssyamm** [comment]
Instagram has tightened the security from last month I guess. All the codes atm are failing to login to the account now. This is how IG trying to protect. All the third party IG service companies have paused their services until they fix the codes. Have to find a way to get into the profile otherwise it would be a dead end.

**Comment by DarkSuniuM** [comment]
Closed all the issues till 2019-01-09, We updated the repo, Please create a new issue if you need! Thanks
